packages/shared/src/modules/friends/libs/enums/friends-api-path.enum.ts

```diff
@@ -2,9 +2,12 @@ const FriendsApiPath = {
 	FOLLOW: "/follow",
 	FOLLOWERS: "/followers",
 	FOLLOWINGS: "/followings",
-	POTENTIAL_FOLLOWING: "/potential-followings",
+	ID: "/:id",
```

Comment:
```suggestion
	$ID: "/:id",
```
for all dynamic params